ohapi/tests/test_test.py

```diff
@@ -29,3 +31,14 @@ def test_load_metadata_csv(self):
         metadata_files = load_metadata_csv('ohapi/tests/data/'
                                            'metadata_proj_file_key_works.csv')
         self.assertEqual(len(metadata_files.keys()), 2)
+
+    def test_strip_zip_suffix(self):
```

Comment:
I think this test is already implicitly integrated by https://github.com/OpenHumans/open-humans-api/blob/master/ohapi/tests/test_test.py#L14. It checks that `.gz` and `.bz2` suffixes are stripped and that the corresponding uncompressed file types are still recognized. (the test coverage evaluation on code climate seems to agree that these lines are already covered). 

Do you think it's necessary to keep this test around?

Reply:
Yeah we can remove that. Thanks for reviewing. I'll update it soon along with adding more tests. :)